integrations/ansible/ansible-tower-remediation/sensu-integration.yaml

```
@@ -102,14 +102,6 @@ spec:
       title: Success
     - type: markdown
       body: |
-        You have successfully enabled the Ansible Tower remediation integration.
+        Success: You enabled the Ansible Tower integration.
 
-        Checks configured with the `ansible-tower` pipeline _and_ remediation actions (i.e. `io.sensu.ansible.config.actions` annotations) will be processed by Ansible Tower on [[ ansible_tower_host ]].
-
-        ```yaml
-        spec:
-          pipelines:
-            - api_version: core/v2
-              type: Pipeline
-              name: ansible-tower
-        ```
\ No newline at end of file
+        Ansible Tower will process events from checks that include the the `io.sensu.ansible.config.actions` remediation action annotation _and_ `ansible-tower` pipeline on [[ ansible_tower_host ]].
```

```suggestion
        Ansible Tower will process events from checks that include the the `io.sensu.ansible.config.actions` remediation action annotation _and_ `ansible-tower` pipeline on [[ ansible_tower_host ]].
        
        Add the `ansible-tower` pipeline reference to one or more checks:
        
         ```yaml
        spec:
          pipelines:
            - api_version: core/v2
              type: Pipeline
              name: ansible-tower
        ```
```